fmriprep/workflows/fieldmap/phdiff.py

```diff
@@ -75,7 +75,7 @@ def _pick1st(inlist):
     magmrg = pe.Node(IntraModalMerge(), name='magmrg')
 
     # de-gradient the fields ("bias/illumination artifact")
-    n4 = pe.Node(ants.N4BiasFieldCorrection(dimension=3), name='n4')
+    n4 = pe.Node(ants.N4BiasFieldCorrection(dimension=3, copy_header=True), name='n4')
```

Comment:
Note that this wasn't using `init_n4bias_wf` already. If there's a good reason for not copying the header, let me know and I'll revert this.